bot/__init__.py

```diff
@@ -168,6 +171,46 @@ def _get_word(self) -> str:
         result = self.buffer[self.previous:self.index + (pos+2)]
         self.index += 2
 
+    # Check if a command in the form of `bot.tags['ask'] {= 'whatever'}` was used
+    elif current == "[":
+        # Helper function to remove any characters we don't care about
+        def clean_argument(arg: str) -> str:
+            return arg.strip("[]'\" ")
+
+        # Syntax is `bot.tags['ask']` => mimic `getattr`
+        if self.buffer.endswith("]"):
+            # Key: The first argument, specified `bot.tags[here]`
+            key = clean_argument(self.buffer[self.index:])
+            # note: if not key, this corresponds to an empty argument
+            #       so this should throw / return a SyntaxError ?
+            args = f"\"{key}\""
+            # Use the cog's `get` command.
+            result = self.buffer[self.previous:self.index] + ".get"
+
+        # Syntax is `bot.tags['ask'] = 'whatever'` => mimic `setattr`
+        elif "=" in self.buffer and not self.buffer.endswith("="):
+            equals_pos = self.buffer.find("=")
+            # Key: The first argument, specified `bot.tags[here]`
+            key = clean_argument(self.buffer[self.index:equals_pos])
+            # Value: The second argument, specified after the `=`
+            value = (clean_argument(self.buffer.split("=")[1])
+                     .replace("\"", "\\\"")  # escape any unescaped quotes
+                     .replace("'", "\\'"))   # to mimic triple quote behaviour.
+            # Use the cog's `set` command.
+            result = self.buffer[self.previous:self.index] + ".set"
+            args = f"\"{key}\" \"{value}\""
```

Comment:
Can `f"\"{key}\" \"{value}\""` be `f'"{key}" "{value}"'` so you won't have to escape any quotes

Reply:
I thought you were part of some cult of only using double quotes, that's why